frontend/src/v4/routes/viewerGui/components/panelsMenu/panelsMenu.component.tsx

```diff
@@ -15,13 +15,13 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import CompareIcon from '@mui/icons-material/Compare';
-import TreeIcon from '@mui/icons-material/DeviceHub';
-import GroupsIcon from '@mui/icons-material/GroupWork';
-import GisIcon from '@mui/icons-material/Layers';
-import ViewsIcon from '@mui/icons-material/PhotoCamera';
-import IssuesIcon from '@mui/icons-material/Place';
-import RisksIcon from '@mui/icons-material/Warning';
+import CompareIcon from '@assets/icons/outlined/compare-outlined.svg';
```

Comment:
I was wondering why we were importing all of these again instead of importing VIEWER_PANEL_ICONS when I realised this component does not use the tickets or drawings icons. These new cards use a different card header which has spacing that is inconsistent with the older cards. The title font size is also different. We should probably make an issue to make these consistent

![image](https://github.com/3drepo/3drepo.io/assets/62481250/5c21aee1-f609-48d8-b6e5-493fc86c3010)

Reply:
there's already one